submissions/fiyingabriel/htmlForms/contact form.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
@Fiyin985 You have created this file in [7] task. No need for it here. You can delete this file for this branch.